Assets/Scripts/Commands/RepaintStrokeCommand.cs

```diff
@@ -13,53 +13,76 @@
 // limitations under the License.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 namespace TiltBrush
 {
     public class RepaintStrokeCommand : BaseCommand
     {
-        private Stroke m_TargetStroke;
-        private Color m_StartColor;
-        private Guid m_StartGuid;
-        private Color m_EndColor;
-        private Guid m_EndGuid;
-        private float m_StartSize;
-        private float m_EndSize;
+        private List<Stroke> m_TargetStrokes;
+        private List<Color> m_StartColors;
+        private List<Guid> m_StartGuids;
+        private List<float> m_StartSizes;
+        private List<Color> m_EndColors;
+        private List<Guid> m_EndGuids;
+        private List<float> m_EndSizes;
```

Comment:
I do wonder if it might be better to create a POD struct that contains the stroke, start & end colors, guids, and sizes and then just have a single list of that type, but I doubt it makes very much difference.

Reply:
Yeah. I was following the style in other commands but there is a point where it's cumbersome. Still bearable at this stage.